src/ot_orchestration/dags/config/unified_pipeline.py

```diff
@@ -105,26 +124,62 @@ def init_pis_config(self) -> dict[str, Any]:
 
         return pis_raw_conf
 
+    def pis_env_vars(self, step_name: str) -> dict[str, str]:
+        """Return the environment variables for a PIS step."""
+        return {
+            "PIS_STEP": step_name.replace("pis_", ""),
+            "PIS_CONFIG_FILE": "/config.yaml",
+            "PIS_POOL": str(self.pis_pool),
+        }
+
+    def ontoform_args(self, step_name: str) -> list[str]:
```

Comment:
Don't you think it belongs better here with the rest of ontoform-config related utilities?

Reply:
It can stay (although similar method for gentropy exists in utils, so it is easy to find it if needed to be reused anywhere else). But definately does not need to be an instance method, you do not call `self`